src/bootstrap/index.js

```diff
@@ -36,6 +50,7 @@ const setup = (
   }
 
   const staticRouter = express.Router();
+  staticRouter.use(protect);
```

Comment:
Looking at the documentation for `overload-protection`, I don't see anything applying to Express Routers.

Should this instead be applied to the first instance of an express server that is created? e.g. https://github.com/govuk-one-login/ipv-cri-common-express/blob/main/src/bootstrap/middleware/index.js#L48C1-L49C27
```js
    // create new express app
    const app = express();
```

Reply:
I originally tried that, which is what was in the readme for overload-protection. However when following that approach overload-protection never takes over, even when max event loop delay is set to 1ms we never actually see any 503s being returned when they should. 

Attaching to the routers as done above does return 503s

Reply:
@merlinc This mismatch in functionality vs documentation is accepted for now, this feature can be investigated in the future if required.